config/example-extras.cfg

```diff
@@ -601,6 +601,31 @@
 #   to not scale the 'wiper_x' parameters.
 
 
+# Configure an SX1509 I2C to GPIO expander. Due to the delay incurred
+# by I2C communication you should NOT use SX1509 pins as stepper enable,
+# step or dir pins or any other pin that requires fast bit-banging. They
+# are best used as static or gcode controlled digital outputs, hardware-pwm
+# pins for e.g. fans (don't use them for heaters) or endstops. When using the
+# pins as endstops, only one axis using an SX1509 pin can be homed at
+# a time. This makes SX1509 endstops NOT suitable for, e.g. delta printers.
+# One may define any number of sections with an "sx1509" prefix. Each expander
+# creates a set of 16 pins (sx1509_my_sx1509:PIN_0 to sx1509_my_sx1509:PIN_15)
+# which can be used elsewhere in the printer configuration.
+#[sx1509 my_sx1509]
+#address:
+#   I2C address used by this expander. Depending on the hardware jumpers
+#   this is one out of the following addresses: 0x3E 0x3F 0x70 0x71. This
+#   parameter must be provided
+#bus: 0
+#   If the I2C implementation of your microcontroller supports multiple I2C
+#   busses, you may specify the bus number here. The default is 0.
+#int_pin:
```

Comment:
Minor, but I wonder if irq_pin or interrupt_pin would be better.  (When I see "int" I think integer.)